### PR TITLE
Feast: thankyou  app download component

### PR DESCRIPTION
**Pull request by paul-daniel-dempsey**
## What are you doing in this PR?

Create/extend Thankyou Module `appDownload`  to display the following thankyou component  if ab-test `feast` enabled (see screenshot).

[**Trello Card**](https://trello.com/c/bEW3vMd7/858-ty-page-component-prompting-download-of-news-app-and-feast)

## Screenshots

image

### CODE REVIEW

File: support-frontend/assets/components/thankYou/thankYouModule.tsx [hunk 1]
```diff
@@ -93,14 +110,46 @@ const bodyContainer = css`
...
+
 const bodyCopyStyle = css`
 	${body.small()};
-	margin-top: ${space[3]}px;
 	margin-bottom: ${space[1]}px;
+	${from.tablet} {
+		font-size: 17px;
+	}
 
+	> h2 {
```
Comment (jamesgorrie) [inline support-frontend/assets/components/thankYou/thankYouModule.tsx:138]: Could we not just apply this CSS to the `h2`? This question goes for other uses of the child-combinator.

File: support-frontend/assets/components/thankYou/thankYouModule.tsx [hunk 2]
```diff
@@ -64,6 +66,21 @@ const downloadAppGridContainer = css`
...
+	grid-template-areas:
+		'icon header'
+		'body body';
+
+	${from.tablet} {
+		grid-template-areas:
+			'icon header'
+			'---- body';
+	}
+`;
```
Comment (jamesgorrie) [inline support-frontend/assets/components/thankYou/thankYouModule.tsx:82]: Fab

**Review by jamesgorrie — APPROVED** [review]
This looks good.

I wonder if the thankYouModule abstraction is making things more complicated than they need to be? Could be worth thinking of abstracting it out to make it easier to use in future.

File: support-frontend/assets/components/thankYou/thankYouModule.tsx (hunk 2 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/components/thankYou/thankYouModule.tsx:82]: Thanks

Comment (paul-daniel-dempsey) [reply support-frontend/assets/components/thankYou/thankYouModule.tsx:82]: Thanks

Comment (paul-daniel-dempsey) [reply support-frontend/assets/components/thankYou/thankYouModule.tsx:82]: Thanks

File: support-frontend/assets/components/thankYou/thankYouModule.tsx (hunk 1 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/components/thankYou/thankYouModule.tsx]: Yep, can see this type of styling is difficult to follow, have removed this `> h2` and another `>  div` further down now too.

**Comment by paul-daniel-dempsey** [comment]
@jamesgorrie  : Also added the Feast App Url placeholders for Android and IOS now.

**Comment by paul-daniel-dempsey** [comment]
> This looks good.
> 
> I wonder if the thankYouModule abstraction is making things more complicated than they need to be? Could be worth thinking of abstracting it out to make it easier to use in future.

Perhaps a ticket for the backlog to overhaul/simplify the whole thankyou setup am guessing?

**Pull request merged**